docs/obis-observer/types.md

```diff
@@ -3,11 +3,32 @@
 Commands use many specific data types packed to bytes.
 This document describes these types and structuring principles.
 
+# Meter profile
 
-## Short name
+Meter profile contains all data which we need to observe, archive and report the OBIS content.
+- archive settings. 
```

Comment:
either from upper letter or not a sentence

Reply:
did it

Reply:
fixed